backend/code-execution-service/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM node:20-alpine
+
+WORKDIR /code-execution-service
+
+COPY package*.json .
+
+RUN npm ci
+
+COPY . .
+
+EXPOSE 3003
```

Comment:
is it suppose to be 3004? since `3004:3004` is in the docker compose file 😬

Reply:
Fixed!